mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -262,6 +265,7 @@ export class ExperimentPage extends Component {
       nextPageToken={this.state.nextPageToken}
       handleLoadMoreRuns={this.handleLoadMoreRuns}
       loadingMore={this.state.loadingMore}
+      key={this.searchRunsRequestId}
```

Comment:
Specify the search request ID as a `key` so that the component is reconstructed on each search, as recommended by the react docs ([link](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key))